src/components/Modals/NewCourse/NewSelfCheckCourseModal.vue

```diff
@@ -88,7 +92,7 @@ export default defineComponent({
   methods: {
     closeCurrentModal() {
       this.reset();
-      this.$emit('close-course-modal');
+      this.$emit('update:modelValue', false);
       // @ts-expect-error: TS cannot understand $ref's component.
       this.$refs.modalBodyComponent.resetDropdowns();
```

Comment:
NewSelfCheckCourseModal: Is kind of buggy. I can't seem to add a course using the modal. There seems to be an issue with dropdowns. Also, when I click outside, I get a TypeError.

https://user-images.githubusercontent.com/54298311/117292225-f2856a80-ae3d-11eb-9d28-123eae6d25a0.mov


![Screen Shot 2021-05-06 at 7 36 02 AM](https://user-images.githubusercontent.com/54298311/117292033-b6eaa080-ae3d-11eb-8dd6-442a9e261a7a.png)